### PR TITLE
Add time CI method

### PR DESCRIPTION
**Pull request by Beraliv**
`npm run time` and `yarn time` are available, `yarn time` is recalculating performance test-like for union (see function `unionTime`), `TIMES.md` is added, `yarn time` shows table with result and rewrite `TIMES.md`, comparison is carried out for `1000` elements, the mean is calculating `1000` times to reduce side effects of laptop and computer. The performance test for union is tested for `3` cases: `0`, `50` and `100%` of elements are the same.

The data is generated in function unionTime, other methods should be placed same way. Maybe use `*.json` for this reason.
`node -v: 8.9.4`
`npm -v: 5.6.0`

### CODE REVIEW

File: misc/cli/time.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  }
+
+  const sets = [
+    // 50% common elements
+    [ big1, big2 ],
+    // no common
+    [ big1, big3 ],
+    // all common
+    [ big1, big1 ]
+	]
```
Comment (Beraliv) [inline misc/cli/time.js]: something wrong with indents

File: misc/cli/time.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    }
+    // Check time
+    methods = methods.map(m => ({
+      name: m,
+      ...getTimes(m)
+    }))
+    // Sort alphabetically
+    methods.sort((a, b) => a.name.toLowerCase().localeCompare(b.name.toLowerCase()))
+    // Create table
+    const header = '## Nanoutils method\'s time'
```
Comment (BabkinAleksandr) [inline misc/cli/time.js]: Maybe empty line after title?

**Comment by Beraliv** [comment]
Ready, please accept and merge @Kelin2025

**Comment by Kelin2025** [comment]
Yes, sir!

**Comment by Beraliv** [comment]
Connected to commits: https://github.com/nanoutils/nanoutils/commit/5b2409c519c96812eda7f8e1b45d8db070d2f3d6 https://github.com/nanoutils/nanoutils/commit/b04af1cdd04f433d5345de10634938a04a6cb0a8 https://github.com/nanoutils/nanoutils/commit/cf3ae36adb92583070e62f2c85a0be8db3683b40

**Pull request merged**